### DISCUSSION TITLE
How to export metrics at xxxxx:6060/metrics

### DISCUSSION THREAD

**Discussion by caoshanhuyi** [blockscout/blockscout#10710] [Q&A]
![image](https://github.com/user-attachments/assets/1b89916d-5103-4f69-9b28-5fee232c5ecf)
sh -c "bin/blockscout eval \"Elixir.Explorer.ReleaseTasks.create_and_migrate()\" && bin/blockscout start"

I started the service using this command, and then I configured the environment variable like this: export PORT=8080. After that, I can access metrics information at xxxxxxxx:8080/metrics. Now, I want to open the metrics separately on port 6060, like this: xxxxxxxx:6060/metrics. Is it possible to achieve this?

**Comment by GimluCom** [comment]
Duplicate of #10720 
Please close this.
